composer-lock-diff

```diff
@@ -24,8 +24,13 @@ if ($opts['md']) {
     ));
 }
 
-print tableize('Production Changes', $prod, $table_opts);
-print tableize('Dev Changes', $dev, $table_opts);
+if (!$opts['dev-only'] || $opts['no-dev']) {
```

Comment:
Looks like "No dev" check is not needed there. If flag "dev only" is passed - we don't need to show production changes.
```suggestion
if (!$opts['dev-only']) {
```